### PR TITLE
replaced how_to with faq

### PR DESCRIPTION
**Pull request by thinksanky**
## Description ##
For some reason my  previous broken links fixes were not reflected.
Trying one more time to replace the how_to to faq.

### CODE REVIEW

File: docs/build_version_doc/build_doc.sh [hunk 1]
```diff
@@ -70,6 +70,14 @@ then
     cp -a "$local_build/." "$web_folder"
 fi
 
+# Date Added: 01/26/2018
+# Feature:    If a change is requested on the asf-site (stable/root/live version), do the following:
+#             1. Checkout the latest branch (eg: 1.0.0)
+#             2. Update
+# 
+
+if
```
Comment (eric-haibin-lin) [inline docs/build_version_doc/build_doc.sh]: Is this `if` intended?

Comment (szha) [reply docs/build_version_doc/build_doc.sh]: @thinksanky ping

File: docs/tutorials/embedded/wine_detector.md [hunk 2]
```diff
@@ -37,9 +37,7 @@ To complete this tutorial, you need:
...
+The first step is to get MXNet with the Python bindings running on your Raspberry Pi 3. There is a tutorial for that provided [here](http://mxnet.io/insstall/index.html). The linked tutorial walks you through downloading the dependencies, and building the full MXNet library for the Pi with the ARM specific compile flags. Be sure to build the library with open CV as we will be using a model that requires it to process images. Then you will register the Python bindings to MXNet. After this is done you should test that your installation works by opening a python REPL on your Pi and typing the following commands:
```
Comment (eric-haibin-lin) [inline docs/tutorials/embedded/wine_detector.md]: `insstall ` -> `install `

File: docs/community/index.md [hunk 3]
```diff
@@ -8,9 +8,9 @@ We track bugs and new feature requests in the MXNet Github repo in the issues fo
 ## Contributors
 MXNet has been developed and is used by a group of active community members. Contribute to improving it! For more information, see [contributions](http://mxnet.io/community/contribute.html).
 
-Please join the contributor mailing list. [subscribe]('mailto:dev-subscribe@mxnet.incubator.apache.org') [archive](https://lists.apache.org/list.html?dev@mxnet.apache.org)
+Please join the contributor mailing list. [subscribe](mailto://dev-subscribe@mxnet.incubator.apache.org) [archive](https://lists.apache.org/list.html?dev@mxnet.apache.org)
```
Comment (eric-haibin-lin) [inline docs/community/index.md]: We need to validate this on the website

**Comment by eric-haibin-lin** [comment]
Merging it for now so that we can get updated nightly test results.

**Pull request merged**